bin/proving-service/README.md

```diff
@@ -85,6 +85,47 @@ The status check returns:
 - The types of proofs the worker supports (transaction, batch, or block proofs)
 - The version of the worker
 
+### Proxy Status Endpoint
+
+The proxy service exposes a status endpoint that provides information about the current state of the proxy and its workers. This endpoint can be accessed at `http://<proxy_host>:<status_port>/status`.
+
+The status endpoint returns a JSON response with the following information:
+- `version`: The version of the proxy service
+- `supported_proof_types`: The types of proofs supported by the proxy
+- `total_workers`: The total number of workers registered with the proxy
```

Comment:
I think I'd disaggregate metrics as much as possible (ie, I wouldn't include `total_workers` since it's derived from `workers`), but maybe something different was decided.